openwisp_users/api/locale/it/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,81 @@
+# OpenWISP
+# Copyright (C) 2021
+# This file is distributed under the same license as the PACKAGE package.
+# Federico Capoano support@openwisp.io, 2021.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-11-03 16:47+0530\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: permissions.py:28
+msgid "Organization not found, `organization_field` not implemented correctly."
+msgstr "Organizzazione non trovata, `organization_field` non implementato correttamente."
+
+#: permissions.py:37
+msgid ""
+"View's permission_classes not implemented correctly.Please use one of the "
+"child classes: IsOrganizationMember, IsOrganizationManager or "
+"IsOrganizationOwner."
+msgstr ""
+"Le classi di autorizzazione della vista non sono state implementate correttamente. Utilizza uno dei "
+"classi figlie: IsOrganizationMember, IsOrganizationManager o "
+"IsOrganizationOwner."
+
+#: permissions.py:46
+msgid ""
+"User is not a member of the organization to which the requested resource "
+"belongs."
+msgstr ""
+"L'utente non è un membro dell'organizzazione a cui la risorsa richiesta "
+"appartiene."
+
+#: permissions.py:56
+msgid ""
+"User is not a manager of the organization to which the requested resource "
+"belongs."
+msgstr ""
+"L'utente non è un manager dell'organizzazione a cui la risorsa richiesta"
+"appartiene."
+
+#: permissions.py:66
+msgid ""
+"User is not a owner of the organization to which the requested resource "
+"belongs."
+msgstr ""
+"L'utente non è un proprietario dell'organizzazione a cui la risorsa richiesta "
+"appartiene."
+
+#: serializers.py:227
+msgid "This field may not be blank."
+msgstr "Questo campo non può essere vuoto."
+
+#: serializers.py:395
+msgid "The two password fields didn’t match."
```

Comment:
same here

Reply:
Can you also double check if a similar msgid is already used in Django and in that case switch to the exact form used in Django? That would also allow to get rid of the translations.

Reply:
Yes, it's registered. https://github.com/django/django/blob/4816dc942860caf076c7c85ea9dbfa8bfab212ff/django/contrib/auth/forms.py#L397-L399